client/src/features/users/Auth/Steps/CreateOrgForm.tsx

```diff
@@ -0,0 +1,293 @@
+import {
+  Button,
+  FormHelperText,
+  Grid,
+  LinearProgress,
+  Typography,
+  ClassNameMap,
+} from '@mui/material';
+import CheckIcon from '@mui/icons-material/Check';
+
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import React from 'react';
+import { Select, TextField } from '../../../../assets/sharedComponents/Forms';
+import { Organization } from '../../../../types';
+import { useMutation, useQuery } from 'react-query';
+import axios, { AxiosResponse } from 'axios';
+import SimpleSnackbar from '../../../action/assets/SimpleSnackbar';
+import { green } from '@mui/material/colors';
+
+const classifications = [
+  { value: 'charitable', text: 'Charitable Organization' },
+  { value: 'religious', text: 'Religious Organization' },
+  { value: 'private', text: 'Private Foundation' },
+  { value: 'political', text: 'Political Organizations' },
+  { value: 'other', text: 'Other' },
+];
+
+const orgValidationSchema = Yup.object().shape({
+  doing_business_as: Yup.string().required('Required'),
+  city: Yup.string().required('Required'),
+  state: Yup.string().required('Required').min(2).max(2),
+  ein: Yup.string()
+    .matches(/^[0-9]\d?-\d{7}$/, 'EIN must match: 99-9999999')
+    .required('Required'),
+  name: Yup.string(),
+  description: Yup.string().required('Required'),
+  website: Yup.string().required('Required'),
+  address: Yup.string().required('Required'),
+  phone: Yup.string()
+    .matches(/^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$/, 'EIN must match: 99-9999999')
+    .required('Required'),
+  nonprofit_classification: Yup.string().min(2).required('Required'),
+});
+
+interface Props {
+  triggerNextStep: (step: number) => void;
+  setParentOrg: (createdOrg: Organization) => void;
+  parentOrg: Organization;
+  classes: ClassNameMap<'button' | 'header' | 'arrow' | 'sideImg' | 'signUpContainer'>;
+}
+const CreateOrgForm = ({
+  setParentOrg = () => {},
+  parentOrg,
+  triggerNextStep = () => {},
+  classes,
+}: Props) => {
+  const [org, setOrg] = React.useState<Organization>(parentOrg);
+  const [triggerEinSearch, setTriggerEinSearch] = React.useState<boolean>(false);
+
+  const orgCreateMutation = useMutation<AxiosResponse<any, any>, Error, Organization, Error>(
```

Comment:
this mutation should be moved to parent and be called at same time (or right after) user create request